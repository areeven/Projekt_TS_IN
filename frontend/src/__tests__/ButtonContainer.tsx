import {render} from '@testing-library/react'
import styled from 'styled-components'

const Button = styled.button`
  color: red
`

test('render button with container', async () => {
    const {container} = render(<Button/>)
    expect(container.firstChild).toMatchSnapshot()
})
