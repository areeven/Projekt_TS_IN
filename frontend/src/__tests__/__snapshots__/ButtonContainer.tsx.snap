// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render button in deleteUser 1`] = `
<button
  class="sc-bdvvtL gcYKbd"
/>
`;

exports[`render button with container 1`] = `
<button
  class="sc-bdvvtL gcYKbd"
/>
`;
